res/controllers/common-controller-scripts.js

```diff
@@ -16,13 +16,45 @@
        Control:off
        Deck:off
  */
+/* exported LibraryColumns */
+/*eslint no-var:off */
+
+// ----------------- Mapping constants ---------------------
+
+// Library column value, which can be used to interact with the CO for "[Library] sort_column"
+var LibraryColumns = {
```

Comment:
I'm sorry I should've been more specific, please put them near the `script.*RegEx` constants. Also, since they're constants, I'd make them CAPS. Also `Object.freeze`.

```javascript
script.LIBRARY_COLUMNS = Object.freeze({
    ARTISTS: 1,
```